src/Catty/Parser&Serialializer/New Parser & Serializer/Serializer/CBXMLSerializer.m

```diff
@@ -56,6 +56,7 @@ - (id)initWithPath:(NSString*)path fileManager:(FileManager *)fileManager
 + (GDataXMLDocument*)xmlDocumentForProject:(Project*)project
 {
     CBXMLSerializerContext *context = [CBXMLSerializerContext new];
+    context.project = project;
```

Comment:
This should be passed directly to the initializer (once `new` is made unavailable this should not be possible anymore).